ludwig/modules/loss_modules.py

```diff
@@ -140,6 +145,37 @@ def call(self, y, y_pred):
         return loss
 
 
+# For Sequence Output Feature
+class SequenceSampledSoftmaxCrossEntropyLoss(tf.keras.losses.Loss):
+    def __init__(
+            self,
+            dec_dense_layer=None,
+            dec_num_layers=None,
+            num_classes=0,
+            feature_loss=None,
+            name=None
+    ):
+        super(SequenceSampledSoftmaxCrossEntropyLoss, self).__init__(name=name)
+
+        self.num_classes = num_classes
+        self.feature_loss = feature_loss
+        self.dec_dense_layer = dec_dense_layer
+        self.dec_num_layers = dec_num_layers
```

Comment:
I believe this parameter is not useful anymore now that you have the `PROJECTION_INPUT` right?

Reply:
removed dec_num_layers parameter